monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ResultsFolderController.java

```diff
@@ -61,12 +70,33 @@
                     resultsFolder.getFiles(null, FileFilters.ANALYSIS_RESULT_SER.getExtension());
             for (final RepositoryFile file : files) {
                 final Map<String, String> map = new HashMap<>();
-                map.put("filename", file.getName());
+                final String name = file.getName();
+                // if the timestamp is not null then we check if we can add the job,
+                // otherwise we continue with loop
+                if (searchedTimestamp != null) {
+                    if (!isEligibleJob(searchedTimestamp, name)) {
```

Comment:
Not really important, but this is pretty weird to me. Why the reverse logic instead of adding the result when isEligibleJob is true?